src/h5p/H5P.util.ts

```diff
@@ -1,4 +1,44 @@
 import { H5PObject } from "../../H5P";
+import { TopicMapItemType } from "../types/TopicMapItemType";
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const H5P = (window as any).H5P as H5PObject;
+
+/**
+ * Replace relative paths to content (image, sound and video files) in
+ * Topic Map Item(s) with absolute paths
+ *
+ * @param items An array with Topic Map Items being used in the current H5P
+ * @param contentId Content id of the H5P being shown
+ * @returns {void}
+ */
+export const addAbsolutePathToFiles = (
+  items: Array<TopicMapItemType> | undefined,
+  contentId: string,
+): void => {
+  if (!items) return;
+
+  items.map(item => {
```

Comment:
Remember to return the mapped array with the changes. `Array#map` will not change in place, but rather return an updated copy.

```suggestion
export const addAbsolutePathToFiles = (
  items: Array<TopicMapItemType> | undefined,
  contentId: string,
): Array<TopicMapItemType> | undefined => {
  if (!items) return;

  return items.map(item => {
```